.github/workflows/mariadb.yml

```diff
@@ -0,0 +1,126 @@
+      - name: Remove bundled Mroonga
+        run: |
+          rm -rf mariadb/extra/groonga
+          rm -rf mariadb/storage/mroonga
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/extra/groonga
+          repository: groonga/groonga
+          submodules: recursive
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/storage/mroonga
+          repository: mroonga/mroonga
+          submodules: recursive
+      - name: Build MariaDB
+        # To reduce build time, we build only the MariaDB core and Mroonga.
+        # The following options disable all other storage engines, module-type
+        # plugins, and some core options to reduce compile time and external
+        # dependencies.
+        run: |
+          cmake \
+            -Smariadb \
+            -Bmariadb.build \
+            -G Ninja \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DCONC_WITH_UNITTEST=OFF \
+            -DMYSQL_MAINTAINER_MODE=OFF \
+            -DPLUGIN_ARCHIVE=NO \
+            -DPLUGIN_AUDIT_NULL=NO \
+            -DPLUGIN_AUTH_ED25519=NO \
+            -DPLUGIN_AUTH_TEST_PLUGIN=NO \
+            -DPLUGIN_AUTH_0X0100=NO \
+            -DPLUGIN_BLACKHOLE=NO \
+            -DPLUGIN_COLUMNSTORE=NO \
+            -DPLUGIN_CONNECT=NO \
+            -DPLUGIN_DAEMON_EXAMPLE=NO \
+            -DPLUGIN_DEBUG_KEY_MANAGEMENT=NO \
+            -DPLUGIN_DIALOG_EXAMPLES=NO \
+            -DPLUGIN_DISKS=NO \
+            -DPLUGIN_EXAMPLE=NO \
+            -DPLUGIN_EXAMPLE_KEY_MANAGEMENT=NO \
+            -DPLUGIN_FEDERATED=NO \
+            -DPLUGIN_FEDERATEDX=NO \
+            -DPLUGIN_FILE_KEY_MANAGEMENT=NO \
+            -DPLUGIN_FTEXAMPLE=NO \
+            -DPLUGIN_FUNC_TEST=NO \
+            -DPLUGIN_HANDLERSOCKET=NO \
+            -DPLUGIN_INNOBASE=NO \
+            -DPLUGIN_LOCALES=NO \
+            -DPLUGIN_METADATA_LOCK_INFO=NO \
+            -DPLUGIN_OQGRAPH=NO \
+            -DPLUGIN_PASSWORD_REUSE_CHECK=NO \
+            -DPLUGIN_PERFSCHEMA=NO \
+            -DPLUGIN_QA_AUTH_INTERFACE=NO \
+            -DPLUGIN_QA_AUTH_SERVER=NO \
+            -DPLUGIN_QA_AUTH_CLIENT=NO \
+            -DPLUGIN_QUERY_CACHE_INFO=NO \
+            -DPLUGIN_QUERY_RESPONSE_TIME=NO \
+            -DPLUGIN_ROCKSDB=NO \
+            -DPLUGIN_SEQUENCE=NO \
+            -DPLUGIN_SERVER_AUDIT=NO \
+            -DPLUGIN_SIMPLE_PASSWORD_CHECK=NO \
+            -DPLUGIN_SPHINX=NO \
+            -DPLUGIN_SPIDER=NO \
+            -DPLUGIN_SQL_ERRLOG=NO \
+            -DPLUGIN_TEST_SQL_DISCOVERY=NO \
+            -DPLUGIN_TEST_SQL_SERVICE=NO \
+            -DPLUGIN_TEST_VERSIONING=NO \
+            -DPLUGIN_TYPE_TEST=NO \
+            -DPROVIDER_LZ4=NO \
+            -DWITH_EMBEDDED_SERVER=NO \
+            -DWITH_UNIT_TESTS=OFF \
+            -DWITH_MARIABACKUP=OFF \
+            -DWITH_SAFEMALLOC=OFF \
+            -DWITH_WSREP=OFF \
+            -DWITHOUT_DYNAMIC_PLUGIN=ON
+          cmake --build mariadb.build
```

Comment:
Could you split `cmake ...` and `cmake --build` into separated steps for easy to see logs?

Reply:
fix: [9c2c697](https://github.com/mroonga/mroonga/pull/896/commits/9c2c69716221e89fd8ea51d4b3763f9d19daad95) Sure.